ui/nuxeo-pagination-controls.js

```diff
@@ -65,8 +65,10 @@ import { html } from '@polymer/polymer/lib/utils/html-tag.js';
               text-align: center;
             }
 
-            --paper-input-container: {
-              padding: 0;
+            --nuxeo-select-dropwdown-menu-padding: 7px 0 0 0;
```

Comment:
Instead of introducing public API, couldn't we try to align the items inside `nuxeo-pagination-controls` instead? Tried quickly with this and they _seemed_ aligned:
```
paper-icon-button, .total {
  padding: 6px 0 10px 0;
}
```

Reply:
This actually doesn't solve our problem. I suggesting another fix without introducing a new public API.

Reply:
After further investigation, we detected that there were some places where our default typography styles were being overridden. I'm suggesting yet another solution that needs the following PR to work correctly: https://github.com/nuxeo/nuxeo-web-ui/pull/1098